pareto/utilities/visualize.py

```diff
@@ -0,0 +1,130 @@
+#####################################################################################################
+# PARETO was produced under the DOE Produced Water Application for Beneficial Reuse Environmental
+# Impact and Treatment Optimization (PARETO), and is copyright (c) 2021-2024 by the software owners:
+# The Regents of the University of California, through Lawrence Berkeley National Laboratory, et al.
+# All rights reserved.
+#
+# NOTICE. This Software was developed under funding from the U.S. Department of Energy and the U.S.
+# Government consequently retains certain rights. As such, the U.S. Government has been granted for
+# itself and others acting on its behalf a paid-up, nonexclusive, irrevocable, worldwide license in
+# the Software to reproduce, distribute copies to the public, prepare derivative works, and perform
+# publicly and display publicly, and to permit others to do so.
+#####################################################################################################
+
+import networkx as nx
+import matplotlib.pyplot as plt
+
+
+def plot_network(m):
```

Comment:
I'd recommend adding an option named "save" or possibly "filename" to this function to allow the user to save the figure that is generated. I think it makes sense for the default behavior to be that the file is not saved, but it would be nice to have this option available.

Reply:
Thats a nice suggestion. I shall add that.

Reply:
Resolved.